src/sentry/consumers/__init__.py

```diff
@@ -394,12 +393,17 @@ def get_stream_processor(
         ) from e
 
     strategy_factory_cls = import_string(consumer_definition["strategy_factory"])
-    logical_topic = consumer_definition["topic"]
```

Comment:
This variable was previously called logical topic, but that was misleading. It was always actually the physical topic which could be overridden via settings.